### PR TITLE
Added Discover View

### PR DESCRIPTION
**Pull request by sbelete01**

### CODE REVIEW

File: client/src/components/Disover.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  const [actorsList, setActorsList] = useState([]);
+  const [movieList, setMovieList] = useState([]);
+
+
+
+  const handleNextClick = () => {
+    getRandomMovie();
+  }
+  const getGenresList = () => {
+    axios('http://localhost:3000/api/genres/')
```
Comment (badams92) [inline client/src/components/Disover.tsx:31]: I'm pretty sure you can remove the `http://localhost:3000` from the URL

File: client/src/components/Disover.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      getRandomMovie()
+        
+    }, [])
+    return (
+      <div>
+        Disover component
+        {/* {console.log(genresList)}
+        {console.log(directorsList)}
+        {console.log(actorsList)} */}
+        {console.log(movieList)}
```
Comment (badams92) [inline client/src/components/Disover.tsx:95]: Try to remove `console.log()` after you finish testing your component

File: client/src/Routes.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import Home from "./components/Home";
 import SearchMovie from "./components/movieSearch";
 import Twitter from "./components/Twitter";
 import Profile from './components/Profile';
-
+import Discover from "./components/Disover";
```
Comment (carlbenjaminlyon) [inline client/src/Routes.tsx:5]: I see that the title of the other file is the same, do you want to keep this, or change it to Discover.tsx?

File: client/src/components/Disover.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+  const handleNextClick = () => {
+    getRandomMovie();
+  }
+  const getGenresList = () => {
+    axios('http://localhost:3000/api/genres/')
+      .then((data) => {
+        const result = data.data.map((movie: any) => movie.genre);
+        setGenresList(result)
+      })
```
Comment (badams92) [inline client/src/components/Disover.tsx:35]: Add a catch block to all of these axios requests for error handling

File: client/src/components/Disover.tsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  const [movieList, setMovieList] = useState([]);
+
+
+
+  const handleNextClick = () => {
+    getRandomMovie();
+  }
+  const getGenresList = () => {
+    axios('http://localhost:3000/api/genres/')
+      .then((data) => {
```
Comment (badams92) [inline client/src/components/Disover.tsx:32]: you can destructure all of these then blocks by using `({data})`

File: client/src/components/Disover.tsx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+      //     setMovieList(data.data)
+      //     const randomNum = Math.floor(Math.random() * 250);
+      //     setCurrentMovie(data.data[randomNum])
+      //   })
+      getRandomMovie()
+        
+    }, [])
+    return (
+      <div>
+        Disover component
```
Comment (badams92) [inline client/src/components/Disover.tsx:91]: Did you mean to leave this in?

File: client/src/components/Disover.tsx (hunk 1 above)
Comment (carlbenjaminlyon) [reply client/src/components/Disover.tsx:31]: Agreed. You can run `axios.get('/api/genres/)`, which will help later so you won't need to go back and change each of these to 'https' when we get to that point.

**Pull request closed without merging**